src/Sylius/Bundle/ReviewBundle/Validator/Constraints/UniqueUserEmail.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ReviewBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @author Mateusz Zalewski <mateusz.p.zalewski@gmail.com>
+ */
+class UniqueUserEmail extends Constraint
+{
+    /**
+     * @var string
+     */
+    public $message = 'sylius.review.author.already_exists';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validatedBy()
+    {
+        return 'unique_user_email_validator';
```

Comment:
We prefix these with ``sylius`` -> ``sylius_unique_reviewer_email_validator`` although I am not sure why would we need this validation... If someone uses this bundle without Sylius, we should be able to comment more than once and it should assign the same reviewer according to e-mail.

Turns out that such bundles cannot be developed properly decoupled unless we start from integration tests containing sample implementation, which is not related to Sylius.